certified-connectors/Tikit/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2652 @@
+    }
+  },
+  "host": "app.tikit.ai",
+  "basePath": "/api/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "multipart/form-data"
+  ],
+  "produces": [],
+  "paths": {
+    "/Ticket": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get all tickets",
+        "description": "Fetch all Tickets",
+        "operationId": "GetAllTickets",
+        "parameters": [
+          {
+            "name": "$expand",
+            "in": "query",
+            "description": "Define oData expand",
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "x-ms-summary": "$expand"
+          },
+          {
+            "name": "$select",
+            "in": "query",
+            "description": "Define oData select",
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "x-ms-summary": "$select"
+          },
+          {
+            "name": "$orderby",
+            "in": "query",
+            "description": "Define oData orderby",
+            "type": "string",
+            "x-ms-visibility": "advanced",
+            "x-ms-summary": "$orderby"
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "description": "Number Tickets",
+            "type": "integer",
+            "default": 10,
+            "enum": [
+              10,
+              25,
+              100,
+              500
+            ],
+            "x-ms-summary": "$top"
+          },
+          {
+            "name": "$skip",
+            "in": "query",
+            "description": "Define oData skip",
+            "type": "integer",
+            "x-ms-visibility": "advanced",
+            "x-ms-summary": "$skip"
+          },
+          {
+            "name": "$count",
+            "in": "query",
+            "description": "Define oData count",
+            "type": "boolean",
+            "x-ms-visibility": "advanced",
+            "x-ms-summary": "$count"
+          }
+        ]
+      }
+    },
+    "/AddTicket": {
+      "post": {
+        "summary": "Create New Ticket",
+        "description": "Create New Ticket",
+        "operationId": "CreateTicket",
+        "parameters": [
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServiceDesk.Core.Actions.AddTicket"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response from AddTicket",
+            "schema": {
+              "$ref": "#/definitions/ServiceDesk.Core.Models.Ticket"
+            }
+          },
+          "default": {

```

Comment:
@TikitTeam Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

Reply:
updated